folder_utils.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func checkYearFolder(year string, path string) string {
+
+	folderInfo, err := os.Stat(path + year)
+	if os.IsNotExist(err) {
+		err := os.Mkdir(path+year, 0755)
+		if err != nil {
+			log.Fatal(err)
+			return "false"
+		}
+
+		return path + year + "/"
+
+	}
+	fmt.Printf(folderInfo.Name())
+	return path + year + "/"
+
+}
+
+func checkCWFolder(cw string, path string) string {
```

Comment:
nimm doch enfach das CW struct hier

Reply:
Mache ich jetzt teilweise. Dein cw.Path() ist aber to much. Da werden Year und Week schon direkt dran gesetzt. Und ich finde es besser die Prüfung einzeln zu machen.